### PR TITLE
Desktop styling

### PR DESCRIPTION
**Pull request by mumo-esther**
Added desktop version media query to my portfolio

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,817 @@
...
+
+  .header-nav-desktop {
+    position: relative;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-end;
+    padding: 0;
+    padding-top: 1em;
+    margin-left: 500px;
+  }
```
Comment (zieeco) [inline style.css]: - Hey, please, let's try to adjust our styles so the nav can be centered as seen in the Figma t make our design close. see the image below:


*Figma design*
Screenshot from 2023-01-19 18-06-11

*your design*
Screenshot from 2023-01-19 18-05-18

File: style.css [hunk 2]
```diff
@@ -0,0 +1,817 @@
...
+
+.works {
+  display: grid;
+  grid-template-columns: repeat(3, auto);
+  padding: 1.5rem;
+  text-align: center;
+  column-gap: 1rem;
+  width: 90%;
+  margin: 0 auto;
+}
```
Comment (zieeco) [inline style.css]: - Hey, Please, let's kindly adjust our styles as your card are having too much space between each other compared to the Figma design. see the images below:

*Figma design*
Screenshot from 2023-01-19 18-14-35


*your design*
Screenshot from 2023-01-19 18-13-50

File: style.css [hunk 3]
```diff
@@ -0,0 +1,817 @@
...
+    content: none;
+  } */
+
+  .about-myself {
+    background:
+      url("./images/frame5.png") no-repeat 0% 20%,
+      url("./images/frame4.png") no-repeat 100% 0%;
+    background-size: 200px, 200px;
+  }
+
```
Comment (zieeco) [inline style.css:725]: - Hey, let's increase the margin-top on this section to add some space as seen in the Figma design below:


*your design*
Screenshot from 2023-01-19 18-20-38

*Figma design*
Screenshot from 2023-01-19 18-20-05

File: style.css [hunk 4]
```diff
@@ -0,0 +1,817 @@
...
+html,
+body {
+  margin: 0;
+  padding: 0;
+  padding-bottom: 2em;
+
+  --shadow: 0 2px 10px -1px rgb(0 0 0 / 80%);
+  --shadow-accent: 0 2px 50px -1px rgb(0 0 0 / 80%);
+  --color-primary: #36b37f;
+}
```
Comment (zieeco) [inline style.css]: :point_right: *This change is applicable to all the sections in your mobile version*

- Hey, your design is having a horizontal scroll on the mobile version and this has made your design inconsistent compared to the Figma design. Please, let's try to fix this issue. See the image below:

*your design*
Screenshot from 2023-01-19 18-37-07

*Figma design*
Screenshot from 2023-01-19 18-39-57

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @mumo-esther ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi @mumo-esther, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Good UI design :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**